### ISSUE TITLE
What's a good way to temporarily disabled plugins?

### ISSUE THREAD

**Issue by AckslD** [wbthomason/packer.nvim#306]
Perhaps more a question/discussion, feel free to let me know if there is a better forum for this than opening an issue :)

Sometimes when having an issues or just working on a plugin I want to have a minimal vim config to see what's going on. Any ideas on how to best do this. What I currently do is to `rm ~/.config/nvim/plugin/packer_compiled.vim` and `mv ~/.local/share/nvim/site/pack ~/.local/share/nvim/site/pack.bak` which feels highly suboptimal.

I do know about `nvim -u NONE` which also does not load plugins, however, in some cases I actually do want to load plugins, for example the plugin I'm currently working on. I've tried to do `nvim -u NORC` and set the `runtimepath`, but things in `~/.local/share/nvim/site/pack` seem to still be loaded.

**Comment by akinsho** [comment]
@AckslD there's actually a discussion section in the repo that this would be a good candidate for but in answer I've also hit this issue with the compiled file being loaded when sometimes I don't want it to be, usually when trying to use a minimal init.lua to debug something.

Generally I delete it which is a pain as you mention, but sometimes I use `--noplugin` which seems to stop things in the `plugin` dir from being read, and then manually loading the plugin I'm debugging by using `packadd` or sourcing it manually

**Comment by AckslD** [comment]
Thanks for your reply @akinsho, always very helpful! :) I missed the discussion section, whoever has access, feel free to move this to there.

I think `packadd` was indeed what I was looking for, I will try this out thanks!

**Comment by yujinyuz** [comment]
I'm currently doing this

 ```lua
use {'romgrk/nvim-treesitter-context', cond = function() return false end}
```

**Comment by wbthomason** [comment]
Yeah, this is a shortcoming of the native packages feature vs. traditional rtp-mangling. Setting `opt = true` and recompiling is probably easiest as long as you can **not** trigger any lazy-loaders; `cond = false` works too, as will @akinsho's trick.
